client/app/cloud/project/compute/infrastructure/virtualMachine/add/cloud-project-compute-infrastructure-virtualMachine-add.html

```diff
@@ -189,7 +189,7 @@
                                                 data-on-click="$ctrl.updateSshKeyRegion()"
                                                 data-ng-if="$ctrl.model.sshKey && ($ctrl.model.region.disabled === 'SSH_KEY' || (region.dataCenters.length === 1 && region.dataCenters[0].disabled === 'SSH_KEY'))">
                                     </oui-button>
-                                    <span data-translate="'cpcivm_add_step2_disabled_' + region.disabled"
+                                    <span data-text="{{ ::'cpcivm_add_step2_disabled_' + region.disabled | translate }}"
```

Comment:
You should've keep `data-translate` => it's a `span` element :)

Reply:
there is an issue with data-translate and string concatenation. Its not working. As an alternative, I have used data-text.

Reply:
@radireddy I think in order to work you have to put your string concatenation in `{{`
```suggestion
                                    <span data-translate="{{ ::'cpcivm_add_step2_disabled_' + region.disabled }}"
```

Reply:
@marie-j , not it did not work, UI shows "cpcivm_add_step2_disabled_"